components/style/properties/gecko.mako.rs

```diff
@@ -1586,6 +1586,29 @@ fn static_assert() {
             T::local => structs::NS_STYLE_IMAGELAYER_ATTACHMENT_LOCAL as u8,
         }
     </%self:simple_image_array_property>
+
+    <%self:simple_image_array_property name="blend_mode" shorthand="background" field_name="mBlendMode">
```

Comment:
There are only a couple image array keyword properties so I don't think auto generation is necessary.